targets/pna_nic/tests/utils.cpp

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2024 Marvell Technology, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+/*
+ * Rupesh Chiluka (rchiluka@marvell.com)
```

Comment:
If you didn't write any line of code in the file, I don't think there is much sense in putting your name

Reply:
Yes. Sorry. Will not repeat it again.